testing/cli/test_cli_sct_get_centerline.py

```diff
@@ -31,5 +33,54 @@ def test_sct_get_centerline_output_file_exists_with_o_arg(tmp_path, ext):
     ensure that the correct output file is created either way."""
     sct_get_centerline.main(argv=['-i', 't2s/t2s.nii.gz', '-c', 't2s', '-qc', str(tmp_path),
                                   '-o', os.path.join(tmp_path, 't2s_centerline'+ext)])
-    for file in [os.path.join('t2s', 't2s_centerline.nii.gz'), os.path.join('t2s', 't2s_centerline.csv')]:
+    for file in [os.path.join(tmp_path, 't2s_centerline.nii.gz'), os.path.join(tmp_path, 't2s_centerline.csv')]:
         assert os.path.exists(file)
+
+
+@pytest.mark.sct_testing
+@pytest.mark.usefixtures("run_in_sct_testing_data_dir")
+@pytest.mark.parametrize('ext', ["", ".nii.gz"])
+def test_sct_get_centerline_soft_output_file_exists(tmp_path, ext):
+    """This test checks the output soft centerline using default usage of the CLI script."""
+    sct_get_centerline.main(argv=['-i', 't2/t2_seg-manual.nii.gz', '-method', 'fitseg', '-centerline-soft', '1', '-o',
+                                  os.path.join(tmp_path, 't2_seg_centerline_soft'+ext), '-qc', str(tmp_path)])
+    for file in [os.path.join(tmp_path, 't2_seg_centerline_soft.nii.gz'), os.path.join(tmp_path, 't2_seg_centerline_soft.csv')]:
+        assert os.path.exists(file)
```

Comment:
I think we could maybe remove this test?

```suggestion
```

My thinking here is:

- The other two tests for `-centerline-soft` will load the `.nii.gz` output file.
- This means that the only unique behavior this test checks for is the existence of the `.csv` file.
- But, this is *basically* covered by the previous tests, too? 
- (All that `-centerline-soft` does is modify the internals of `centerline/core.py` -- it doesn't make many changes to `sct_get_centerline` itself. So the method for generating CSV files is basically the same with or without `-centerline-soft`.)
- Therefore, this test is basically redundant with the previous tests? If one passes, then the other will too.